setup.py

```diff
@@ -19,7 +19,7 @@
 
 
 install_requires = [
-    "Django>=3.2,<4.3",
```

Comment:
Is the top pin really needed? 🤔

It stops folks installing Django 5.0 (or would be 5.1) even though there's no actual incompatibility…

> wagtail 5.2 requires Django<4.3,>=3.2, but you have django 5.0 which is incompatible.

If it just had a minimum it wouldn't cause this problem.

Reply:
Thanks @carltongibson!

I'm in favour of removing it (and upper bounds in general), but we've been doing this for years so I think this needs a discussion with other core team members before we proceed with it.

As for getting a release out, I looked into it this morning and I think I'd like to get a fix for #11300 in. Although Matt's on holiday so I'm not sure if we can get things reviewed and ready for a release before early next week.

Reply:
OK, thanks Sage. No stress. 

Turns out pip only warns and tests pass so it's not a blocker 😜

Thanks for all your work! 🎁

Reply:
I've dropped the upper Django version in c1e468e.

Thank you for taking the time to review this, Carlton

Reply:
No problem! Thanks (again) for the work! 🎁